### PR TITLE
Update rule description

### PR DESCRIPTION
**Pull request by cpandhi**
updates to the rule description to include scope and expand on the bullets are ready for review.

### CODE REVIEW

File: act-framework.bs [hunk 1]
```diff
@@ -42,10 +42,11 @@ Rule Description {#structure-description}
 
 Each ACT Rule MUST have a description that:
 
-* Is in plain language;
-* Explains the accessibility requirement being tested;
-* Lists any limitations and assumptions for the test;
-* Explains the failure conditions and their reasons.
+* Is in plain language and provides a brief description of what the rule does.
+* Explains the accessibility requirement being tested such as the WCAG SC or the technique the rule maps to; For example WCAG 2.0 Technique H67.
+* Lists any limitations and assumptions for the test, the test environment, technologies being used or the subject being tested; For example: A rule for 1.4.1: Use of Color has to make an assumption with CSS-properties are used to make a link visually evident. Typically something like background, border, color, font, or text-decoration.
```
Comment (maryjom) [inline act-framework.bs]: Editorial: '...has to make an assumption **that** CSS properties', don't hyphenate CSS properties, change 'with' to 'that'.  The last sentence is missing a verb, so suggest saying 'Typical examples are: background, border, color, font, or text decoration'.  Text decoration should not be hyphenated.

File: act-framework.bs [hunk 2]
```diff
@@ -42,10 +42,11 @@ Rule Description {#structure-description}
...
+* The scope or the context of the rule;
```
Comment (maryjom) [inline act-framework.bs]: Need a verb for this to be consistent.  'Contains the scope or the context of the rule;'

**Comment by moekraft** [comment]
This duplicates much of the Rule Outline. The Rule Description should be part of the Rule Outline and in plain language describe what the rule does..

**Comment by moekraft** [comment]
See PR #43

File: act-framework.bs [hunk 3]
```diff
@@ -42,10 +42,11 @@ Rule Description {#structure-description}
...
 
-* Is in plain language;
-* Explains the accessibility requirement being tested;
-* Lists any limitations and assumptions for the test;
-* Explains the failure conditions and their reasons.
+* Is in plain language and provides a brief description of what the rule does.
+* Explains the accessibility requirement being tested such as the WCAG SC or the technique the rule maps to; For example WCAG 2.0 Technique H67.
+* Lists any limitations and assumptions for the test, the test environment, technologies being used or the subject being tested; For example: A rule for 1.4.1: Use of Color has to make an assumption that CSS properties are used to make a link visually evident. Typical examples are: background, border, color, font, or text decoration.
+* Provides the scope or the context of the rule; 
+* Explains the failure conditions and their reasons. For example the rule triggers on all images on the page that has no alt attribute defined.
```
Comment (rdeltour) [inline act-framework.bs]: > the rule triggers o all the images on the page that has no alt attribute defined

do it mean the rule is applied to all such images, or returns a `fail` to all such images? (in other words, is it an example of the rule scope and context, or of the rule failure condition?)

Comment (moekraft) [inline act-framework.bs]: This duplicates much of the Rule Outline. The Rule Description should be part of the Rule Outline and in plain language describe what the rule does..

**Comment by cpandhi** [comment]
Have pushed changes based on review.

File: act-framework.bs [hunk 4]
```diff
@@ -42,10 +42,8 @@ Rule Description {#structure-description}
 
 Each ACT Rule MUST have a description that:
 
-* Is in plain language;
-* Explains the accessibility requirement being tested;
-* Lists any limitations and assumptions for the test;
-* Explains the failure conditions and their reasons.
+* Is in plain language and provides a brief description of what the rule does.
+* Explains the accessibility requirement being tested such as the WCAG SC or the technique the rule maps to; For example WCAG 2.0 Technique H67.
```
Comment (nitedog) [inline act-framework.bs]: If I recall correctly, a rule always applies to at least one success criterion, and may map to any number of techniques? If so, should this be separated into two points:
 - Lists the accessibility requirement being tested, such as the WCAG Success Criterion
 - Refers to any related information, such as the WCAG Techniques it maps to

File: act-framework.bs [hunk 5]
```diff
@@ -42,10 +42,8 @@ Rule Description {#structure-description}
 
 Each ACT Rule MUST have a description that:
 
-* Is in plain language;
-* Explains the accessibility requirement being tested;
-* Lists any limitations and assumptions for the test;
-* Explains the failure conditions and their reasons.
+* Is in plain language and provides a brief description of what the rule does.
```
Comment (rdeltour) [inline act-framework.bs]: "a rule must have a description that provides a brief description of what the rule does" sounds a bit tautological 😉.
I'd suggest "provides a summary" or "provides a short explanation" instead.

File: act-framework.bs (hunk 4 above)
Comment (cpandhi) [inline act-framework.bs]: Good point Shadi, i will make the suggested changes. Thx.

File: act-framework.bs (hunk 5 above)
Comment (cpandhi) [inline act-framework.bs]: Like short explanation, will make the change.

**Comment by cpandhi** [comment]
Have pushed updates based on review comments.

**Pull request merged**